tree/ntuple/inc/ROOT/RPageStorage.hxx

```diff
@@ -818,6 +819,10 @@ public:
    // TODO(gparolini): for symmetry with SealPage(), we should either make this private or SealPage() public.
    RResult<ROOT::Internal::RPage>
    UnsealPage(const RSealedPage &sealedPage, const ROOT::Internal::RColumnElementBase &element);
+
+   // Builds the streamer info records from the descriptor's extra type info section. This is necessary when
+   // connecting streamer fields so that emulated classes can be read.
```

Comment:
```suggestion
   /// Builds the streamer info records from the descriptor's extra type info section. This is necessary when
   /// connecting streamer fields so that emulated classes can be read.
```